plenary/plenary-26.yaml

```diff
@@ -0,0 +1,211 @@
+metadata:
+  title: Resolutions of the 26th plenary meeting of ISO/TC 154
+  date: 2009-09-25
+  source: Secretary
+resolutions:
+  - dates:
+      - 2009-09-25
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Approval of specification of codelists for TDED/EDED alignment"
+    identifier: 291
+    
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: UNANIMOUS
+
+    actions:
+      - type: resolves
+        date_effective: 2009-09-25
+        message: |
+        ISO/TC 154, having considered the results of TDED Joint MA meeting on 2009-09-24, resolves to approve that the codelists for TDED / EDED alignment should include: 
+        - UN/EDIFACT Codelist - UNCL
+        ⎯ UNECE Trade Recommendation Codelists – currently maintained by ICG
+        And that ISO 9735 service elements and recommendations are not included.
```

Comment:
I'm not sure about this part... Is it correct?
Original source reference:
![capture3](https://user-images.githubusercontent.com/33627611/71758847-431c3a80-2e7b-11ea-9759-d11281230c5d.JPG)

Reply:
I've updated it. It was missing "considerations:".